### PR TITLE
add Automation time tracking

### PR DESCRIPTION
**Pull request by Johannes-Thiel**
There is a Timetracker. But it seams to be used for Time on a specific field.
I made a tracker to track all the time in automations. This Tacker works for all weeding implements and the Recorder. It can be seen in the status drawer.
image

### CODE REVIEW

**Comment by Johannes-Thiel** [comment]
At the moment the time is in seconds, we should talk about the correct format of the data.

**Comment by Johannes-Thiel** [comment]
The tracking was moved to the base implement. The unnecessary Kpis were removed and are no longer shown in the UI.
The working Hours can be seen in the Status drawer on the right side of the UI.

File: field_friend/automations/implements/implement.py [hunk 1]
```diff
@@ -42,3 +50,17 @@ def settings_ui(self):
...
+            return
+        if self.start_time is None:
+            self.start_time = rosys.time()
+        passed_time = rosys.time() - self.start_time
+        if passed_time > 1:
+            self.system.kpi_provider.increment_all_time_kpi('time')
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
```
Comment (pascalzauberzeug) [inline field_friend/automations/implements/implement.py:66]:
```suggestion
            self.start_time = rosys.time()
```
What happened here? 😄

**Review by pascalzauberzeug — CHANGES_REQUESTED** [review]
I think we should talk about what we really want to track. Going by these changes the time when changing rows won't be tracked because the implements are deactivated during that.

File: field_friend/automations/implements/implement.py (hunk 1 above)
Comment (Johannes-Thiel) [reply field_friend/automations/implements/implement.py:66]: I Have no idea 😄

Comment (Johannes-Thiel) [reply field_friend/automations/implements/implement.py:66]: i dont know.

**Comment by Johannes-Thiel** [comment]
maybe we could put this tracking into the navigation?

**Review by LukasBaecker — CHANGES_REQUESTED** [review]

**Review by LukasBaecker — APPROVED** [review]

**Pull request merged**